### PR TITLE
A bunch of updates to logind and resolved man pages

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: man/systemd-resolved.service.xml [hunk 1]
```diff
@@ -119,98 +119,97 @@
...
-      these days, it's generally recommended to avoid defining <literal>.local</literal> in a DNS server, as
-      <ulink url="https://tools.ietf.org/html/rfc6762">RFC6762</ulink> reserves this domain for exclusive
+      <listitem><para>Names for which synthetic records are generated (the local hostname,
+      <literal>localhost</literal> and <literal>localdomain</literal>, local gateway, as listed in the
+      previous section) and addresses configured in <filename>/etc/hosts</filename> are never routed to the
+      network and a reply is sent immediately.</para></listitem>
+
+      <listitem><para>Single-label names are resolved using LLMNR on all local interfaces where LLMNR is
+      enabled. Lookups for IPv4 addresses are only sent via LLMNR on IPv4, and lookups for IPv6 addresses are
+      only sent via LLMNR on IPv6. Note that lookups for single-label synthetised names are not routed to
```
Comment (poettering) [inline man/systemd-resolved.service.xml]: usually we use the z spelling with `synthesize`

File: man/systemd-resolved.service.xml [hunk 2]
```diff
@@ -216,6 +218,56 @@
...
+
+    <itemizedlist>
+      <listitem><para>Some names are always resolved internally (see Synthetic Records above). Traditionally
+      they would be resolved by <filename>nss-files</filename>, and only if provided in
+      <filename>/etc/hosts</filename>.</para></listitem>
+
+      <listitem><para>Single-label names are not resolved for A and AAAA records (unless overriden with
+      <varname>ResolveUnicastSingleLabel=</varname>, see
+      <citerefentry><refentrytitle>resolved.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>).
+      </para></listitem>
```
Comment (poettering) [inline man/systemd-resolved.service.xml:239]: they are resolved, just not with unicast DNS. That should be made clear

Comment (poettering) [reply man/systemd-resolved.service.xml:239]: (hmm, might make sense to mention that this is like resolv.conf's `no-tld-query` option being always – at least that's how i undestood that option, never tried it)

File: man/systemd-resolved.service.xml [hunk 3]
```diff
@@ -216,6 +218,56 @@
...
+      servers.</para></listitem>
+
+      <listitem><para>This resolver reads and caches <filename>/etc/hosts</filename> internally. (In other
+      words, <filename>nss-resolve</filename> replaces <filename>nss-files</filename> in addition to
+      <filename>nss-dns</filename>). Entries in <filename>/etc/hosts</filename> have highest priority.</para>
+      </listitem>
+
+      <listitem><para>This resolver also implements LLMNR and MulticastDNS in addition to the classic unicast
+      DNS protocol, and will resolve single-label names using LLMNR (when enabled) and names ending in
+      <literal>.local</literal> using MulticastDNS (when enabled).</para></listitem>
```
Comment (poettering) [inline man/systemd-resolved.service.xml:269]: might also make sense to mention that neither $LOCALDOMAIN nor $RES_OPTIONS are honoured right now (we might want to compat support some of them though)

File: man/systemd-resolved.service.xml (hunk 2 above)
Comment (keszybz) [reply man/systemd-resolved.service.xml:239]: Indeed, I added a sentence about that.

**Comment by keszybz** [comment]
Updated with the suggested changes, nothing else.

**Pull request merged**